cli/config/configuration.toml.in

```diff
@@ -140,13 +140,27 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # Default false
 #use_vsock = true
 
-# VFIO devices are hotplugged on a bridge by default. 
+# VFIO devices are hotplugged on a bridge by default.
 # Enable hotplugging on root bus. This may be required for devices with
-# a large PCI bar, as this is a current limitation with hotplugging on 
+# a large PCI bar, as this is a current limitation with hotplugging on
 # a bridge. This value is valid for "pc" machine type.
 # Default false
 #hotplug_vfio_on_root_bus = true
 
+# A comma-separated list of IOMMU groups to coldplug into the VM.
+# The devices belong to these groups are only coldplugged if the
+# corresponding IOMMU group is passed using --device=/dev/vfio/XX
+# Example: "5, 7, 11"
+pci_coldplug_groups = "@DEFPCICOLDPLUGGROUPS@"
+
+# A list of parameters to pass to the vfio-pci driver when coldplugging
+# devices. This is a semicolon-separated list of device option lists where
+# a device option list is comma-separated list where the first entry is
+# the device's BDF. All devices that will be coldplugged should be added
```

Comment:
I'm finding this a little difficult to read. Could it be changed to something like this without losing the meaning?:

> A semi-colon separated list of parameters to pass to the vfio-pci driver when coldplugging
> devices. Each list element is itself a comma separated list of options which must start with the BDF of the device to be cold-plugged and optionally followed by one or more device-specific options.